store/tikv/kv.go

```diff
@@ -109,6 +110,26 @@ type tikvStore struct {
 	etcdAddrs    []string
 	mock         bool
 	enableGC     bool
+
+	safePoint uint64
+	spTime    time.Time
+	spSession tidb.Session  // this is used to obtain safePoint from remote
+	spMutex   sync.RWMutex  // this is used to update safePoint and spTime
+	spMsg     chan struct{} // this is used to nofity when the store is closed
+}
+
+func (s *tikvStore) CheckVisibility() (uint64, error) {
```

Comment:
This function name is not consistent with its return values.
`CheckXXX` should return bool.